app/index.html

```diff
@@ -13,8 +13,7 @@
       window.location = '/browserUnsupported.html';
     </script>
   <![endif]>
-
-  <script src="scripts/clientSideRedirection.js"></script>
+  -->
```

Comment:
It's supposed to close the tag on line 11. But should I change line 15 to `<![endif]-->`? I'm not familiar with IE conditional comments. The problem is that currently, everything between line 11 and the end of the Adobe DTM comment is being commented out.